pax-manifest/src/utils.rs

```diff
@@ -0,0 +1,130 @@
+use crate::{LiteralBlockDefinition, SettingElement, Token, TokenType, ValueDefinition};
+
+use pest::iterators::Pair;
+use pest::Parser;
+use pest_derive::Parser;
+
+#[derive(Parser)]
+#[grammar = "pax.pest"]
+pub struct PaxParser;
+
+//What do to with location info?
+//a lot of functionality is copied atm
+pub fn to_value_definition(raw_value: &str) -> Option<ValueDefinition> {
```

Comment:
Let's return a Result here with a descriptive error. I'd imagine we can use this as part of the validation for the orm.

Reply:
Agreed! Done :)